### PR TITLE
Passkey iOS example

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Quick. example of how to use the Passkey APIs on iOS. Android should be mostly the same.

## Before you submit

- Please add **unit tests** where it makes sense to do so

### CODE REVIEW

File: languages/swift/iOS/App/ContentView.swift [hunk 1]
```diff
@@ -320,6 +332,40 @@ struct ContentView: View {
...
+            clientDataHash: Data(),
+            allowList: nil,
+            options: Options(rk: true, uv: Uv.preferred),
+            extensions: nil
+        ))
+        
+        try await authenticator.silentlyDiscoverCredentials(rpId: "")
+        
+        // Only on android!
+        let client = clientFido.client(userInterface: ui, credentialStore: cs)
```
Comment (dani-garcia) [inline languages/swift/iOS/App/ContentView.swift:364]: Do we want to hide this impl behind a cfg to make sure it's not usable here?

**Review by dani-garcia — DISMISSED** [review]
LGTM, how did you format this code? I couldn't get Xcode to do it for me when I wrote it 😭

**Comment by Hinton** [comment]
Format, `brew install swiftlint, `swiftlint --fix`

File: languages/swift/iOS/App/ContentView.swift (hunk 1 above)
Comment (Hinton) [reply languages/swift/iOS/App/ContentView.swift:364]: Maybe, I think it's better to do it once we fully understand everything though.

**Review by dani-garcia — APPROVED** [review]

**Pull request merged**